clients/privacy-center/app/store.ts

```diff
@@ -85,19 +87,5 @@ export const makeStore = (preloadedState?: Partial<RootState>) => {
   return store;
 };
 
-const store = makeStore();
```

Comment:
I moved creating the Redux store into the App code itself (see _app.tsx). That means it's no longer statically created - instead, it can rely on some runtime props. This is needed because I want to hydrate the Redux store with the config JSON from the server-side environment 👍